aten/src/ATen/native/native_functions.yaml

```diff
@@ -4347,6 +4350,7 @@
     SparseCUDA: hspmm_sparse_cuda
 
 - func: copy_sparse_to_sparse_(Tensor(a!) self, Tensor src, bool non_blocking=False) -> Tensor(a!)
+  device_guard: False
```

Comment:
This calls into `Tensor::to` : https://github.com/pytorch/pytorch/blob/bf37bf7da46aa54476f07b02d1211cb0d1a06b35/aten/src/ATen/SparseTensorUtils.h#L34-L39

which handles device guard: https://github.com/pytorch/pytorch/blob/bf37bf7da46aa54476f07b02d1211cb0d1a06b35/aten/src/ATen/native/TensorConversions.cpp#L83-L85

Reply:
Maybe this should have been a composite haha